src/components/feed/CreateFeed/BasicInformation.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+
+import { Form, FormGroup, TextInput, TextArea } from '@patternfly/react-core';
+import Client, { Tag } from '@fnndsc/chrisapi';
+import { Typeahead } from 'react-bootstrap-typeahead';
+
+export declare var process: { 
+  env: {
+    REACT_APP_CHRIS_UI_URL: string
+  }
+}
+
+interface BasicInformationProps {
+  authToken: string,
+  feedName: string,
+  feedDescription: string,
+  tags: Tag[],
+  handleFeedNameChange: (val: string, ev: React.ChangeEvent<HTMLInputElement>) => void,
+  handleFeedDescriptionChange: (val: string, ev: React.ChangeEvent<HTMLInputElement>) => void,
+  handleTagsChange: (tags: Tag[]) => void,
+}
+
+interface BasicInformationState {
+  availableTagsLoaded: boolean,
+  availableTags: Tag[], 
+}
+
+class BasicInformation extends React.Component<BasicInformationProps, BasicInformationState> {
+  
+  constructor(props: BasicInformationProps) {
+    super(props);
+    this.state = {
+      availableTagsLoaded: false,
+      availableTags: [],
+    }
+  }
+
+  componentWillMount() {
+    this.fetchTagList().then((tags: Tag[]) => {
+      this.setState({
+        availableTagsLoaded: true,
+        availableTags: tags
+      })
+    });
+  }
+
+  async fetchTagList() {
+    const client = new Client(process.env.REACT_APP_CHRIS_UI_URL, { token: this.props.authToken} );
+    await client.getFeeds(); // getFeeds must be called on new Client objects
```

Comment:
Hopefully these lines can now be improved as you mentioned in an earlier comment .

Reply:
Turns out 0.0.6 didn't fix that, but 0.0.8 did. Those lines (and the equivalent in ChrisFileSelect have been removed by passing the instantiated client as a prop.